packages/dita-example-sx-modules-xsd-common-element-mod/src/install.js

```diff
@@ -0,0 +1,28 @@
+import documentsManager from 'fontoxml-documents/src/documentsManager.js';
+import addAction from 'fontoxml-operations/src/addAction.js';
+import remoteDocumentStateManager from 'fontoxml-remote-documents/src/remoteDocumentStateManager.js';
+
+export default function install() {
+	addAction(
+		'isOutOfSyncDocument',
+		function isOutOfSyncDocument(stepData) {},
+		function getIsOutOfSyncDocumentState(stepData) {
+			var documentId = documentsManager.getDocumentIdByNodeId(stepData.contextNodeId);
+
+			// If document out of sync, we need to disable the operation
+			const documentStatus = remoteDocumentStateManager.getState(documentId);
```

Comment:
Won't a similar problem occur when the document becomes read-only for some other reason (e.g., when the lock becomes unavailable)?

Reply:
Then, I suggest to disable the operation when the document doesn't have a status, that is, the document is fine.